community/drupal7/APKBUILD

```diff
@@ -1,15 +1,32 @@
 # Contributor: Carlo Landmeter <clandmeter@gmail.com>
 # Maintainer: Andy Postnikov <apostnikov@gmail.com>
-_php=php5
 pkgname=drupal7
-pkgver=7.60
+pkgver=7.61
 pkgrel=0
 pkgdesc="An open source content management platform"
 url="https://www.drupal.org/"
 arch="noarch"
 license="GPL"
-depends="${_php}-fpm ${_php}-xml ${_php}-json ${_php}-gd ${_php}-ftp ${_php}-posix ${_php}-curl ${_php}-zlib
-	${_php}-dom ${_php}-ctype ${_php}-sockets"
+depends="php7-fpm
+	php7-bcmath
+	php7-ctype
+	php7-curl
+	php7-dom
+	php7-gd
+	php7-gmp
+	php7-iconv
+	php7-json
+	php7-mbstring
+	php7-opcache
+	php7-openssl
+	php7-pcntl
+	php7-pdo_mysql
+	php7-pdo_sqlite
```

Comment:
Actually only one of mysql/pgsql/sqlite needed - but this one are most commonly used